### PR TITLE
Manager Submission View

### PR DESCRIPTION
**Pull request by axd7832**
This page is where a Manager will come to view submitted forms and other submissions. This links up the backend that @gabelandau was working on yesterday.

### CODE REVIEW

File: database/migrations/2019_01_19_204656_create_submissions_table.php [hunk 1]
```diff
@@ -24,8 +24,8 @@ public function up()
             $table->integer('milestone_id')->unsigned()->nullable();
             $table->integer('company_id')->unsigned()->nullable();
 
-            $table->foreign('form_id')->references('id')->on('forms')->onDelete('set null');
-            $table->foreign('company_id')->references('id')->on('companies')->onDelete('set null');
```
Comment (gabelandau) [inline database/migrations/2019_01_19_204656_create_submissions_table.php]: Hopefully this doesn't cause cascade issues later, but we figure out how to deal with it later on if it does.

File: resources/js/pages/manager/ManagerSubmittedForms.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      submissions:[]
```
Comment (gabelandau) [inline resources/js/pages/manager/ManagerSubmittedForms.vue]: You're gonna hate me but can you add a space before the `[]`?

File: database/migrations/2019_01_19_204656_create_submissions_table.php (hunk 1 above)
Comment (axd7832) [reply database/migrations/2019_01_19_204656_create_submissions_table.php]: this is what got it to work locally on my machine, not sure why it was violating a foreign key constraint

File: resources/js/pages/manager/ManagerSubmittedForms.vue (hunk 2 above)
Comment (axd7832) [reply resources/js/pages/manager/ManagerSubmittedForms.vue]: i hate u

**Pull request merged**